docs/06-patterns/navigation.mdx

```diff
@@ -0,0 +1,65 @@
+---
+title: Navigation
+description: Navigation patterns are crucial for ensuring users can seamlessly move through an application.
+hide_title: true
+---
+
+# Navigation <Badge text='ready' color='green'></Badge>
+Navigation patterns are crucial for ensuring users can seamlessly move through an application. In Grafana, two key components play a vital role in navigation: the Megamenu and Breadcrumbs. Let's delve into how these components contribute to effective navigation patterns.
+
+## Megamenu Navigation
+
+**Overview:**  
+The Grafana megamenu is the primary navigation component for all of Grafana, including core, plugins, and other apps. It exists in three states - opened, closed, and “docked” open. When open, the menu acts as an overlay. When docked, the page content responds to a narrowed viewport. 
+
+**When to Use:**  
+The megamenu provides access to levels 1, 2, and 3 of the Grafana information architecture. Levels beyond this should be handled by tabs on a page (level 4) or inline elements using headings, steppers or other design elements (level 5 and up).
+
+**Usage Guidelines:**  
+- Except in rare cases, cloud applications should not be placed at level 1 of the megamenu. Instead, they should be grouped together with similar applications.
+- Ensure all megamenu entries have a corresponding page or place to go for consistent breadcrumb behavior.
+- Level 1 content must have an icon next to it for visual consistency; other levels do not utilize icons.
+- Utilize a landing page for all overview/category pages.
+- Ensure all megamenu entries are in sentence case.
```

Comment:
FMPOV, I'd avoid repeating the rules. For example, I see this point as a "Dos": 'Do ensure all megamenu entries are in sentence case'. I'd leave the "Usage Guidelines" for general indications.

Reply:
Addressed this 👍